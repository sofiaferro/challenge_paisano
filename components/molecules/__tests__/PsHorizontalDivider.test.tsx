import React from 'react';
import renderer from 'react-test-renderer';

import { describe, expect, it } from '@jest/globals';

import PsHorizontalDivider from '../PsHorizontalDividerLarge';

describe('PsHorizontalDivider', () => {
  it('renders the component', () => {
    const tree = renderer.create(<PsHorizontalDivider />);
    expect(tree).toMatchSnapshot();
  });
});
