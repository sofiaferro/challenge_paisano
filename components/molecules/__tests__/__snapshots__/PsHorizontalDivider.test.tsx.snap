// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PsHorizontalDivider renders the component 1`] = `
<div
  className="sc-e5ce1a5b-0 kzTqjN"
>
  <img
    alt="Horizontal divider"
    data-nimg="1"
    decoding="async"
    height={40}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={40}
  />
</div>
`;
